Octokit.Tests/Clients/UserGpgKeysClientTests.cs

```diff
@@ -0,0 +1,119 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using NSubstitute;
+using Xunit;
+
+namespace Octokit.Tests.Clients
+{
+    public class UserGpgKeysClientTests
+    {
+
+        public class TheCtor
+        {
+            [Fact]
+            public void EnsuresNonNullArguments()
+            {
+                Assert.Throws<ArgumentNullException>(() => new UserGpgKeysClient(null));
+            }
+        }
+
+        public class TheGetAllForCurrentMethod
+        {
+            [Fact]
+            public void RequestsTheCorrectUrl()
+            {
+                var connection = Substitute.For<IApiConnection>();
+                var client = new UserGpgKeysClient(connection);
+
+                var expectedUri = ApiUrls.GpgKeys().ToString();
```

Comment:
On the unit tests side, we usually hardcode the expectedUri... as a minor safety net to guard against typos in the `ApiUrls` helper class, or inadvertent edits etc